ElementX/Sources/Screens/RoomScreen/View/TimelineItemList.swift

```diff
@@ -198,6 +180,25 @@ struct TimelineItemList: View {
         return selectedItemId == item.id ? 1.0 : 0.5
     }
     
+    private func updatePinnedOffset(scrollViewProxy proxy: ScrollViewProxy) {
+        guard let item = pinnedItem else {
+            return
+        }
+        
+        if item.animated {
+            withAnimation(Animation.elementDefault) {
+                proxy.scrollTo(item.id, anchor: item.anchor)
+            }
+        } else {
+            proxy.scrollTo(item.id, anchor: item.anchor)
+        }
```

Comment:
Could this be simplified to? Genuinely don't know if it is 100% equivalent or not.
```suggestion
        withAnimation(item.animated ? .elementDefault : .noAnimation) {
            proxy.scrollTo(item.id, anchor: item.anchor)
        }
```

Reply:
Unfortunately they are not, it behaves like it's fully animated and doesn't even scroll all the way down.